CHANGELOG.md

```diff
@@ -5,6 +5,22 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 
 ## Unreleased
 
+## 1.4.0 - 2015-11-18
+
+Updating web type spacing and leading
+
+__Note:__ We are no longer optically adjusting margins/padding to match design mockups pixel perfectly
+
+### Changed
+
+- Updated text leading (line-height) to match recommendation from DM issue #349
```

Comment:
Can we abbreviate this to CFPB Design Manual or include a link to the issue instead? DM is a little jargony.